py/jupyterlite/src/jupyterlite/__init__.py

```diff
@@ -0,0 +1,2 @@
+"""a command line tool for building JupyterLite sites"""
+__version__ = "0.1.0"
```

Comment:
```suggestion
__version__ = "0.1.0a0"
```

Reply:
right, we need to single-source that from... someting, or at least put it under integrity control

Reply:
maybe the version of the tarball it ships

Reply:
normalized this and pyolite (which is also hard-coded). Need a plan... if only an integrity regex check...